src/test/java/uk/gov/pay/ledger/queue/QueueMessageReceiverIT.java

```diff
@@ -20,10 +18,8 @@
 
 public class QueueMessageReceiverIT {
 
-    @RegisterExtension
-    public static AppWithPostgresAndSqsExtension rule = new AppWithPostgresAndSqsExtension(
-            config("queueMessageReceiverConfig.backgroundProcessingEnabled", "true")
-    );
+    @ClassRule
```

Comment:
Can we use `@RegisterExtension` ? ClassRule takes this test back to JUnit4

Reply:
Good spot! I'll update the annotation